dspace-spring-rest/src/test/java/org/dspace/app/rest/CommunityRestRepositoryIT.java

```diff
@@ -26,6 +36,65 @@
 
 public class CommunityRestRepositoryIT extends AbstractControllerIntegrationTest {
 
+    public void createTest() throws Exception {
```

Comment:
we need also tests for the attempt from anonymous and unauthorized users

Reply:
@abollini Will do. Thanks for your review!